endpoint.bal

```diff
@@ -166,22 +169,22 @@ public client class Client {
     # Receive message(s) from the queue
     #
     # + queueResourcePath - Resource path to the queue from the host address. e.g.: /610968236798/myQueue.fifo 
-    # + maxNumberOfMessages - Maximum number of messages returned. Possible values are 1-10. Default is 1
-    # + visibilityTimeout - Duration (in seconds) that messages are hidden from subsequent requests
-    # + waitTimeSeconds - Wait time in seconds
-    # + attributeNames - List of attributes that need to be returned along with each message
-    # + messageAttributeNames -  Name of the message attribute
-    # + receiveRequestAttemptId - Token used for deduplication of receive message calls. only to FIFO queues
+    # + maxNumberOfMessages - (optional) Maximum number of messages returned. Possible values are 1-10. Default is 1
+    # + visibilityTimeout - (optional) Duration (in seconds) that messages are hidden from subsequent requests
+    # + waitTimeSeconds - (optional) Wait time in seconds
+    # + attributeNames - (optional) List of attributes that need to be returned along with each message
```

Comment:
Optionality is given by Ballerina syntax `?`. If we have it in doc comment, we will have to maintain it (Meaning if we change the code, need to change doc comment too). Anyways - better if we move `(optional)` to the end of param description lines.

Reply:
With the changes requested later. (optional) word has been removed from adding in end of the doc comment